src/DynamoCLI/CommandLineRunner.cs

```diff
@@ -61,8 +56,10 @@ private static XmlDocument RunCommandLineArgs(DynamoModel model, StartupUtils.Co
             // If the desire is to have additional states you can add logic here to build
             // up a list and iterate through each state in the list using the loop below.
             // This must be done after potentially loading states from an external file.
-            var stateNames = new List<String>();
-            stateNames.Add("default");
+            var stateNames = new List<String>
```

Comment:
wow, I have not seen this code in a long time - you can get rid of this state nonsense, it was for setting presets, back when those existed.